src/main/resources/templates/search.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>搜索结果</title>
+</head>
+<body>
+<h1>搜索</h1>
+<div id="searchForm">
+    <h2>精确搜索</h2>
+    <form action="/api/search" method="post">
+        <input type="text" name="q" placeholder="Search" aria-label="search"/>
+        <input type="submit" value="Go">
+    </form>
+</div>
```

Comment:
Check about the login template. At the `action` attribute of this form, we might need to use `th:action` or something else to let Spring Security auto insert a CSRF token field.